cmd/juju/commands/switch.go

```diff
@@ -39,8 +39,12 @@ When used without an argument, the command shows the current controller
 and its active model. 
```

Comment:
Can you rejig the usage details so that it is one paragraph? It looks a bit messy just now.